Transaction/DLC_Funding_Transaction.md

```diff
@@ -0,0 +1,110 @@
+# DLC Funding Transaction
+
+This details the exact format of the on-chain funding transaction, which both sides need to agree on to ensure signatures are valid.
+
+# Table of Contents
+
+* [Transaction](#transaction)
+  * [Transaction Input and Output Ordering](#transaction-input-and-output-ordering)
+  * [Funding Inputs](#funding-inputs)
+  * [FundingOutput](#funding-output)
+  * [Change Outputs](#change-outputs)
+* [Fees](#fees)
+
+# Transaction
+
+## Transaction Input and Output Ordering
+
+The Offerer/Initiator's inputs come first and in the order that they were listed in the Offer message, the Accepter's inputs come after this in the order they are listed in the Accept message. The funding output comes first, followed by the Offerer's change output if applicable, and then the Accepter's change output if applicable.
+
+Note that this will likely change in the future.
+
+## Funding Inputs
+
+All funding inputs must be Segwit or nested P2SH(Segwit) in order to protect against malleability attacks.
+
+## Funding Output
+
+* The funding output script is a P2WSH to:
+
+`2 <offer_funding_pubkey> <accept_funding_pubkey> 2 OP_CHECKMULTISIG`
+
+* Where both `pub_key`s are in compressed format.
+
+## Change Outputs
+
+The funding transaction's change outputs should pay to the address specified in the relevant offer/accept message. A change output's value should equal the total funding amount of that party subtracted by their total collateral as well as their fees for both this transaction as well as their fees for the largest possible Contract Execution Transaction. If this value is below the dust limit of `1000 satoshis`, then no change output is added for that party.
+
+# Fees
```

Comment:
We should ensure that funding transaction feerate is above Core's default `minRelayTxFee` (1000sat per kB). Though maybe in contract negotiation at feerate announcement, rejecting frail feerate proposal.